docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,37 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Account 
+
+### Balance
+
+Used to lookup address asset balances
```

Comment:
can you cover whether this includes non-fungible assets?

Reply:
Resolved in https://github.com/WalletConnect/walletconnect-specs/pull/206/commits/393a30b86b9852bea474372754bcae4482fa1e4b